Oblig 2/Oppgave 2.ipynb

```diff
@@ -18,18 +18,29 @@
   },
```

Comment:
<p>Tenker du her mener at forskjellen i ytelse mellom de to modellene er statistisk signifikant fordi p-verdien er så lav? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Bsingstad/DMBA2024/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>